File: app/src/main/java/com/bearcave/passageplanning/tides/database/TideCRUD.java
package com.bearcave.passageplanning.tides.database;

import com.bearcave.passageplanning.base.database.withcustomkey.CRUDWithCustomKey;

import org.joda.time.DateTime;

import java.util.Collection;


public interface TideCRUD extends CRUDWithCustomKey<TideItem, DateTime> {
    void insertAll(Collection<TideItem> tides);
}

File: app/src/main/java/com/bearcave/passageplanning/tides/database/TideCRUD.kt
package com.bearcave.passageplanning.tides.database

import com.bearcave.passageplanning.base.database.withcustomkey.CRUDWithCustomKey

import org.joda.time.DateTime


interface TideCRUD : CRUDWithCustomKey<TideItem, DateTime> {
    fun insertAll(tides: Collection<TideItem>)
}
